TestApp/android/app/build.gradle

```diff
@@ -82,14 +82,6 @@ android {
         versionCode 1
         versionName "1.0"
     }
-    signingConfigs {
```

Comment:
Why you removed it?

Reply:
it depends on debug.keystore which I removed because CI complained.

Reply:
Don't we need debug.keystore anymore?

Reply:
It's just a test app. No need.